parsl/monitoring/web_app/index.py

```diff
@@ -0,0 +1,62 @@
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, Output
+from parsl.monitoring.web_app.app import app, init_db
+import argparse
+
+
+def web_app(db, port):
+    if not init_db(db):
+        return
```

Comment:
I think this means that if the DB file doesn't exist, then the web app doesn't start.

There might be a race condition there when starting from within the DFK, where the logging DB doesn't exist yet? (because it's created in another process that might run ahead or behind this?) I'm not sure.